test/WebJobs.Script.Tests/FunctionMetadataProviderTests.cs

```diff
@@ -214,24 +196,14 @@ public void ValidateFunctionName_DoesNotThrowOnValidName(string functionName)
         {
             try
             {
-                FunctionMetadataManager.ValidateName(functionName);
+                FunctionMetadataProvider.ValidateName(functionName);
             }
             catch (InvalidOperationException)
             {
                 Assert.True(false, $"Valid function name {functionName} failed validation.");
             }
         }
 
-        [Fact]
-        public void ReadFunctionMetadata_Succeeds()
```

Comment:
Where is this test now?

Reply:
I had missed this one. Have added it back here.
https://github.com/Azure/azure-functions-host/pull/4980/files#diff-210d2369bbeb2645e900222fa323617aR43